src/main/resources/com/hcl/appscan/jenkins/plugin/Messages.properties

```diff
@@ -8,6 +8,10 @@
 
 error.checking.results=An error occurred checking the results of analysis. The status of analysis is {0}.
 error.invalid.template.file=Invalid scan template file.
+error.invalid.login.sequence.file= Invalid login sequence file.
+error.login.fields.automatic=AppScan plugin (DAST) - Username and password are required to scan your application.
+error.login.fields.manual=AppScan plugin (DAST) - The login sequence file format is invalid. Only Config file extension is supported.
+error.login.fields.empty.manual = AppScan plugin (DAST) - A login sequence file is required to scan your application.
```

Comment:
I suggest leaving out the "AppScan plugin (DAST) - " portions of these messages. It should be clear what's happening without that.